circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/replica/ReplicaTest.java

```diff
@@ -163,9 +163,6 @@ public void prepare() throws Exception {
   private void convertSourceTableToView() {
     sourceTable.setTableType(TableType.VIRTUAL_VIEW.name());
     sourceTable.getSd().setLocation(null);
-    when(mockSourceLocationManager.getTableLocation()).thenReturn(null);
-    when(mockSourceLocationManager.getPartitionLocations()).thenReturn(Collections.<Path> emptyList());
-    when(mockSourceLocationManager.getPartitionSubPath(any(Path.class))).thenReturn(null);
```

Comment:
`private @Mock SourceLocationManager mockSourceLocationManager;` can also be removed